frontend/src/pages/Overview.js

```diff
@@ -45,10 +89,6 @@ const Overview = () => {
                 <ParentInfo id="parentInfo-container">
                     {!buildUrl ? (
                         <FlexColumn>
-                            <SeriesInfo id="series-info-container">
```

Comment:
I Think the the Title should be still shown even if there series isn't

Reply:
This title was the title of the last build container